crates/youki/src/commands/info.rs

```diff
@@ -223,6 +224,38 @@ pub fn print_namespaces() {
     }
 }
 
+#[inline]
+fn get_cap_supported(caps: &caps::CapsHashSet, cap: caps::Capability) -> &'static str {
+    if caps.contains(&cap) {
+        "available"
+    } else {
+        "unsupported"
```

Comment:
Can we  have either `supported` and `unsupported` or `available` and `unavailable`? I thought on it and it feels it'd be better if we have one of these two